content/docs/views.md

```diff
@@ -0,0 +1,102 @@
+
+Choosing **Post-It** will get rid of the title bar, menu bar, and tool bar (similar to full-screen mode), but it will maintain the same window size as Notepad++ previously took up, rather than enlarging to take up the whole screen.  This mode will also hide the tab bar, so you can focus on working on the single visible tab, though you can still use the tab-switching shortcuts to choose another tab. Like full-screen mode, Post-It mode has the `⊞` in the upper right of the window for leaving this mode, or you can use the shortcut (the default shortcut is `F12`) or exit Notepad++ and restart.
+
+Choosing **Distraction Free Mode** (new to v7.9.6) will combine full-screen mode with Post-It mode, so it will go full screen, and will hide the title bar, menu bar, tool bar, and tab bar; it will also add wide margins within the edit menu, so the text doesn't go so wide across the full-screen window.
+
+## View Current File In...
+
+The **View Current File In...** sub-menu will launch the current file in a standard web browser (Firefox, Chrome, Edge, or IE).  This works best for HTML or other web documents, but text, XML, and other file types can also be viewed in the browser.
+
+## Show Symbol
+
+The sub-menu **Show Symbol** sub-menu will affect whether certain characters are rendered with special glyphs so that they can easily be seen or identified in the editor view, and whether certain tab and wrap guides will be visible. All of the toggles in this sub-menu are **_[Remembered]_**.
+
+There are three entries that are grouped together, which can affect the rendering of certain characters with alternate glyphs. Please note that while the characters may be displayed differently in Notepad++, the underlying file has the normal characters encoded per the current **Language** menu settings.  Only one of these three options can be selected at a time, and selecting a new option will un-select the old.  If you want none of these symbol options, click on the option that is currently selected and it will be turned off.
+
+* **Show Whitespace and TAB**: When selected, the space character will show as a [colored](#global-styles "Style Configurator > Global Styles > White Space Symbol") dot `·`, and the tab as a colored arrow `→` that expands to fill the width of the tab.
+* **Show End of Line**: When selected, the carriage return character will be rendered as `CR` in a small box, the line feed character as `LF`, and a Windows two-character line ending as `CRLF`.

```

Comment:
Add note about how coloring of line-ending representations can't be changed (or can it? how do themes do it), or in general the "intrusiveness" of the line-ending representation can't be reduced ??

Reply:
Technically, it's reverse-video for Global Styles > Default (which is how themes affect it)